source/Cosmos.System2/FileSystem/CosmosVFS.cs

```diff
@@ -38,29 +39,32 @@ public class CosmosVFS : VFSBase
         /// <exception cref="DirectoryNotFoundException">Thrown on fatal error.</exception>
         public override void Initialize()
         {
-            mPartitions = new List<Partition>();
-            mFileSystems = new List<FileSystem>();
-            mRegisteredFileSystems = new List<FileSystemFactory>();
-
-            RegisterFileSystem(new FatFileSystemFactory());
+            if (BlockDevice.Devices.Count == 0)
+            {
+                global::System.Console.WriteLine("No disks found!");
```

Comment:
This should be suppressible, in which case an exception is thrown for the user os to handle.